pkg/ocibundle/tools/loop.go

```diff
@@ -0,0 +1,26 @@
+package tools
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sylabs/singularity/pkg/util/loop"
+)
+
+// CreateLoop ...
+func CreateLoop(file *os.File, offset, size uint64) (string, error) {
```

Comment:
Do you think all SIF specific part should also go here? Here is not much point in having this function that already accepts everything. Instead it can accept `sif *os.File` and then load it and do all necessary stuff to find offset & size that is done in `Create()` now.

Reply:
This function is generic all related stuff to SIF goes into sif_driver